packages/framework/src/Resources/config/admin_menu.yml

```diff
@@ -6,26 +6,20 @@
   route: admin_order_list
   icon: document-copy
   items:
-    - label: 'Orders overview'
-      route: admin_order_list
-      items:
-        - label: 'Editing order'
-          route: admin_order_edit
-          visible: false
+    - label: 'Editing order'
```

Comment:
I wonder if we should consider these items as MenuItem, it is not displayed in menu and href is in template. Is it because of breadCrumb or why?

Reply:
Exactly because of Breadcrumb. I agree it's weird but solving it is out of scope of this redesign.